hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.module.scss

```diff
@@ -0,0 +1,26 @@
+@import "./../../../assets/abstracts/mixins";
```

Comment:
You had a style for your button before and now it's gone? It should just set the color to "primary"